e2e/connection_test.go

```diff
@@ -157,6 +157,9 @@ func consumerConnectFlow(t *testing.T, tequilapi *tequilapi_client.Client, consu
 	assert.Equal(t, connectionStatus.SessionID, se.SessionID)
 	assert.Equal(t, "New", se.Status)
 
+	// Keep active connection for some time to check for statistics change.
+	time.Sleep(5 * time.Second)
```

Comment:
It should be enough to update stats on disconnect, no need to wait 5 seconds.

Reply:
e2e was not testing statistics update loop at all since it was stopping connection immediately. I added it to be at least a bit realistic.